lms/djangoapps/courseware/tabs.py

```diff
@@ -318,6 +320,16 @@ class DatesTab(CourseTab):
     view_name = "dates"
     is_dynamic = True
 
+    def __init__(self, tab_dict):
+        def link_func(course, reverse_func):
```

Comment:
This could just be a class level method, yah? It's fine inline too, but seems a little beefier than typical one-line inline methods or methods that you don't want to expose, so you make them inline.